src/alfasim_sdk/_internal/alfacase/case_description_attributes.py

```diff
@@ -192,6 +192,11 @@ class Foo:
         )
         return Curve(image, domain)
     elif isinstance(value, Curve):
+        if value.GetLength() == 0:
```

Comment:
:eyes:

Reply:
Yes, I will create the task to guarantee that it is being tracked.

Only to track the discussion, here is the `alfasim-sdk` PR link https://github.com/ESSS/alfasim-sdk/pull/297.

Reply:
Task: ASIM-5237

Reply:
Nice, thanks!